fvtr/fvtr.sh

```diff
@@ -42,6 +42,9 @@ function usage () {
 	echo "       tests -  a list of tests to run (optional).  It is going"
 	echo "                to run all tests by default."
 	echo "       Example: fvtr -f ~/advance-toolchain/config.at5.0.sles10 "
+	echo ""
+	echo "   or: fvtr report"
+	echo "       the report subcommand outputs the logs of failed tests"
```

Comment:
When we created this function, we tried to follow the same output style of other programs.
This addition isn't following that style.  It would be interesting if it could follow it.
However, I don't think this change should block this patch.
When you make that change, please explain the difference between -l and report.